packages/forms-web-app/src/controllers/final-comment/comments-question.js

```diff
@@ -38,6 +38,6 @@ exports.postCommentsQuestion = async (req, res) => {
 	req.session.finalComment.hasComments = hasComments;
 
 	return hasComments
-		? res.redirect(VIEW.FINAL_COMMENT.FINAL_COMMENT)
-		: res.redirect(VIEW.FINAL_COMMENT.CHECK_YOUR_ANSWERS);
+		? res.redirect(`/${VIEW.FINAL_COMMENT.FINAL_COMMENT}`)
```

Comment:
If you need those single quotes it'd be better to have then within the constant value rather than having to concatenate them in the constant declaration. That makes things not very tidy IMO

Reply:
understood, but this is consistent with how this is handled in existing controllers and the format of the constants in the list of views at `packages/forms-web-app/src/lib/views.js`